e2e/ansible/playbooks/resiliency/test-k8s-master-failure/k8s-jenkins-pod-vars.yml

```diff
@@ -0,0 +1,30 @@
+---
+test_name: hyperconverged_jenkins_upgrade_on_k8s
```

Comment:
@vibhor995 is this jenkins upgrade test or master failure test?? please modify the test-name if it is master-failure test.

Reply:
yes. it is modified

Reply:
@vibhor995 This file should go to k8s-jenkins folder right?